views/js/previewer/plugins/content/scss/highlighterTray.scss

```diff
@@ -0,0 +1,80 @@
+.highlighter-container {
+    $highlighterBackground: #1F1F1F;
```

Comment:
Mixing variables declarations always look messy. Does it really necessary? Also bear in mind  that  local **CSS** variables can't be used in ie11